src/rdma.c

```diff
@@ -334,7 +334,7 @@ static int rdmaCreateResource(RdmaContext *ctx, struct rdma_cm_id *cm_id) {
     int comp_vector = rdma_config->completion_vector;
 
     if (ibv_query_device(cm_id->verbs, &device_attr)) {
-        serverLog(LL_WARNING, "RDMA: ibv ibv query device failed");
+        serverLog(LL_WARNING, "RDMA: ibv query device failed");
```

Comment:
this and the item below are notable. I don't speak RDMA/ibv. But most of the items in this file don't duplicate `ibv` so I'm guessing this is an error, but I could definitely be wrong.